src/usr/local/www/diag_packet_capture.php

```diff
@@ -479,7 +482,14 @@ function fixup_host($value, $position) {
 	}
 
 	if ($port != "") {
-		$matches[] = "port ".fixup_not($port);
+		for($i = 0; $i < count($ports); $i++){
+                    	 if($i == count($ports)-1) {
```

Comment:
Needs a space between `if` and `(`, between the `)` and `-`, and between `-` and `1`).